content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -12,6 +12,10 @@ weight: 10
 
 Client certificates generated by [kubeadm](/docs/reference/setup-tools/kubeadm/) expire after 1 year. This page explains how to manage certificate renewals with kubeadm.
 
+{{< warning >}}
+By default, kubeadm configures kubelet to renew its certificate automatically in `/var/lib/kubelet/pki/`, and it is not handled by `kubeadm certs` subcommands. The auto-renew process may sometime fail, see [below](#kubelet-client-cert).
+{{< /warning >}}
```

Comment:
i think we can remove this block.

there is already this in the document:
```
{{< note >}} kubelet.conf is not included in the list above because kubeadm configures kubelet for automatic certificate renewal. {{< /note >}}
```

we can amend the same note to link to a new section in here:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/

that seems a better place for the mini-guide despite my initial suggestion. sorry about that.

amended version:
```
{{< note >}} kubelet.conf is not included in the list above because kubeadm configures kubelet for automatic certificate renewal. To troubleshoot an expired kubelet client certificate see [Troubleshooting kubelet client certificates rotation](....) {{< /note >}}
```

Reply:
Ok. I also added also a link to https://kubernetes.io/docs/tasks/tls/certificate-rotation/ if that's ok ?